src/systemverilog/systemverilog.test.ts

```diff
@@ -475,6 +475,129 @@ testTokenization('systemverilog', [
 			{ startIndex: 16, type: ''},
 			{ startIndex: 17, type: 'identifier.sv'}
 		]
-	}]  
+	}],
+
+	// Comments
```

Comment:
Can we also add multi line comments and doxygen comments in our tests

Reply:
Added a few more tests, I don't think we need that many considering they should all just be commented, but what are the thoughts?